### PR TITLE
DOCS-1646: WAF restructuring

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:
(https://tigera.atlassian.net/browse/DOCS-1646)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Review by electricjesus — APPROVED** [review]

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 1]
```diff
@@ -69,7 +71,9 @@ Enabling WAF for certain system services may result in an undesired cluster stat
 
 :::
 
-**Required**
+### Prerequisites
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx:74]:
```suggestion
### Required
```

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 2]
```diff
@@ -79,18 +83,75 @@ modify the `default` FelixConfiguration to set the field `policySyncPathPrefix`
...
+:::note
+
+If WAF is already enabled in your cluster, you do not see this page.
+:::
+
+#### Enable WAF using kubectl CLIs
+
+In the ApplicationLayer custom resource, named `tigera-secure`, set the `webApplicationFirewall` field to `Enabled`.
+
+```bash
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx:123]:
```suggestion
```yaml
```

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 3]
```diff
@@ -103,6 +164,68 @@ kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"policySync
 
 4. To make further changes to a service, click **Actions**, and then **Enable** or **Disable**.
 
+You have now applied WAF rule sets to your own services, and note that the traffic that goes through the selected services will be alerted but not blocked by default.
+
+### Setting WAF rule to block
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]:
```suggestion
### Set WAF rule to block traffic
```

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 4]
```diff
@@ -313,7 +304,7 @@ To disable WAF on a service, use the Actions menu on the WAF board, or use the f
    kubectl annotate svc <service-name> -n <service-namespace> projectcalico.org/l7-logging-
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx]:
```bash
kubectl annotate svc <service-name> -n <service-namespace> projectcalico.org/l7-logging-
```

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 5]
```diff
@@ -79,18 +83,75 @@ modify the `default` FelixConfiguration to set the field `policySyncPathPrefix`
...
+we recommend that you install the [GoogleCloudPlatform/microservices-demo app](https://github.com/GoogleCloudPlatform/microservices-demo):
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/release/v0.7.0/release/kubernetes-manifests.yaml
+```
 ## How to
 
-- [Enable WAF](#enable-waf)
+- [Enable WAF on your cluster](#enable-waf-on-your-cluster)
+- [Apply WAF to your services](#apply-waf)
```
Comment (bmckercher123) [inline calico-enterprise/threat/web-application-firewall.mdx:96]:
```suggestion
- [Apply WAF to your services](#apply-waf-to-your-services)
```

**Review by bmckercher123 — APPROVED** [review]
Fantastic reorg! A few nits.

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx:123]: The entire block can be used as a `bash` command. So, I used `bash` there.

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 3 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 4 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx]: Done

File: calico-enterprise/threat/web-application-firewall.mdx (hunk 5 above)
Comment (girishshankaran) [reply calico-enterprise/threat/web-application-firewall.mdx:96]: Done

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 6]
```diff
@@ -79,18 +83,75 @@ modify the `default` FelixConfiguration to set the field `policySyncPathPrefix`
...
+
+If WAF is already enabled in your cluster, you do not see this page.
+:::
+
+#### Enable WAF using kubectl CLIs
+
+In the ApplicationLayer custom resource, named `tigera-secure`, set the `webApplicationFirewall` field to `Enabled`.
+
+```bash
+cat << EOF | kubectl apply -f -
```
Comment (hjiawei) [inline calico-enterprise/threat/web-application-firewall.mdx:124]: This line is simplified in https://github.com/tigera/docs/pull/860.

File: calico-enterprise/threat/web-application-firewall.mdx [hunk 7]
```diff
@@ -163,157 +286,25 @@ spec:
   description: 'Test WAF Global Alert'
   severity: 1
   dataSet: waf
-  period: 1m
-  lookback: 1h
-  query: '"rule_info" IN {"*942100*"}'
+  period: 5m
+  lookback: 5m
+  query: 'rules.id = "942100"'
```
Comment (hjiawei) [inline calico-enterprise/threat/web-application-firewall.mdx:291]: This change causes a validation error.

**Pull request merged**